ooiui/templates/science/plotting.html

```diff
@@ -462,73 +465,83 @@
     //  
     this.listenTo(this.collections.plotEventList, 'reset', function(collection) {
       self.views.plotEventListView.collection = collection;
+
+
+
       $.when(self.views.plotEventListView.render()).done(function() {
-        $('#eventsTable').fixedHeaderTable({ altClass: 'odd', footer: true, fixedColumns: false, });  
+
+        $("table").stickyTableHeaders({ scrollableArea: $(".scrollable-area"), "fixedOffset": 2 });
```

Comment:
@dlangston no id? were just grabbing the table, im thinking this should have an id...